.github/workflows/ci.yml

```diff
@@ -138,13 +138,12 @@ jobs:
           ROOT=$PWD
           for lib in $(find libraries/aws libraries/standard -maxdepth 1 -mindepth 1)
           do
-            cd $lib
+            cd $ROOT/$lib
             if [ ! -d docs/doxygen/ ]; then
               echo "$lib has no Doxygen documentation."
-              exit 0
+            else
+              doxygen docs/doxygen/config.doxyfile
             fi
-            doxygen docs/doxygen/config.doxyfile
-            cd $ROOT
```

Comment:
If we don;t change back to the ROOT, then the next step, "Run Doxygen on CSDK and verify stdout is empty", won't be running doxygen on the CSDK, it will be running in a library repo?

Reply:
Good point. Fixed the `cd` problem and tested it with a deliberate doxygen issue in 7bc84c0

Reply:
Also thank you for catching my mistake.